[CU-86c03echw] Проект всплывающее меню при нажатии на аккаунт-DEV

poundforpound opened the pull request.

tomfordrumm committed:
ts fix

tomfordrumm committed:
Merge pull request #15 from 404-Hub/CU-86bzv9xfv-dev

[CU-86bzv9xfv] Проект - Раздел “Создание запроса на совместную разраб… - DEV

tomfordrumm committed:
Merge pull request #17 from 404-Hub/CU-86bzv9xfv-dev

[CU-86bzv9xfv] Проект - Раздел “Создание запроса на совместную разраб…- DEV

tomfordrumm committed:
Merge pull request #20 from 404-Hub/CU-86bzzrqqu-dev

[Cu-86bzzrqqu] Аккаунт - страница аккаунта - DEV

tomfordrumm committed:
Merge pull request #22 from 404-Hub/CU-86bzva1h5-dev

[CU-86bzva1h5] Аккаунт - Страница “Профиль” - DEV

MMadiyar417 committed:
при пустых задачах, добавлен текст и кнопка.Приложил три варианта текста

MMadiyar417 committed:
удалены не нужные фрагменты

MMadiyar417 committed:
удалены не нужные фрагменты

MMadiyar417 committed:
Merge branch 'CU-86bzvg1by' into CU-86bzvg1by-dev

tomfordrumm committed:
Merge pull request #24 from 404-Hub/CU-86bzvg1by-dev

[CU-86bzvg1by] Аккаунт - Мои задачи - показывать сообщение пользователя без задач - dev

MMadiyar417 committed:
Merge branch 'main' of github.com:404-Hub/FRONTEND into CU-86bzv9vqm-dev

tomfordrumm committed:
Merge pull request #28 from 404-Hub/CU-86bzvfphh-dev

[CU-86bzvfphh] Проект - Взять на себя - показывать сообщение для гостя - DEV

MMadiyar417 committed:
Merge branch 'CU-86bzv9vqm' into CU-86bzv9vqm-dev

MMadiyar417 committed:
удалена непонятная строка

MMadiyar417 committed:
?

tomfordrumm committed:
Merge pull request #26 from 404-Hub/CU-86bzv9vqm-dev

[CU-86bzv9vqm] Категории - кнопка Загрузить еще - dev

tomfordrumm committed:
Merge pull request #29 from 404-Hub/CU-86c03e1bz-dev

[CU-86c03e1bz] Аккаунт - Баги - DEV

poundforpound committed:
remove console.log

tomfordrumm committed:
Merge pull request #31 from 404-Hub/CU-86bzvxhdr-dev

[CU-86bzvxhdr]-Регистрация - показывать сообщение при повторной регистрации-DEV

MMadiyar417 committed:
добавлена логика возврата на страницу Идеи при отмене создания пати

tomfordrumm committed:
[CU-86bzy7132] Создать сущности Project + переименовать App в Idea

tomfordrumm committed:
[CU-86bzy7132] Создать сущности Project + переименовать App в Idea

tomfordrumm committed:
Merge pull request #34 from 404-Hub/CU-86bzv9x5c-dev

[Cu 86bzv9x5c] - Рейтинг - сохранять голос и показывать сообщение для гостя DEV

MMadiyar417 committed:
реализовано через router

tomfordrumm committed:
Merge pull request #37 from 404-Hub/CU-86c035g6m-dev

[Cu 86c035g6m] Запрос пати - добавить действие для кнопки Назад -  DEV

tomfordrumm commented:
Task linked: [CU-86c03echw Хедер - добавить меню пользователя](https://app.clickup.com/t/86c03echw)

tomfordrumm committed:
[CU-86bzy7132-dev] Merge branch 'dev' into CU-86bzy7132-dev

# Conflicts:
#	src/app/[lang]/ideas/[id]/_components/IdeaDetails.tsx
#	src/app/[lang]/tasks/page.tsx
#	src/components/my-tasks/TasksList.tsx

tomfordrumm committed:
Merge pull request #40 from 404-Hub/CU-86bzy7132-dev

[CU-86bzy7132] Создать сущности Project + переименовать App в Idea - DEV

tomfordrumm commented:
Неправильная последовательность merge
